### DISCUSSION TITLE
CLB will be used for testing in CEM Script project

### DISCUSSION THREAD

**Discussion by euonymos** [mlabs-haskell/clb#14] [Announcements]
Instead of PSM, we are going to use CLB for testing within the next milestone of the [CEM Script project](https://github.com/mlabs-haskell/cem-script) also funded by Catalyst Fund 10.

CEM Script can generate its own off-chain code for building transactions and hence needs exactly the functionality provided by CLB: submitting transactions and running some queries to check the state once they made it to the chain.

We believe this will be beneficial for both projects. CEM Script will benefit from using a very simple and fast fully compatible with the ledger emulator, and CLB  will get feedback from this process which will help form its API.

**Comment by uhbif19** [comment]
I think we may not change CEM milestones, cuz CLB is PSM successor.

**Reply by euonymos** [reply]
Sorry, wdym by "fix CEM milestones"?

**Reply by uhbif19** [reply]
Change them by change request.

**Reply by euonymos** [reply]
I see, yep, there is no need to change anything in the Catalyst project.
